R/expr-expr.R

```diff
@@ -1512,7 +1512,7 @@ expr__arg_unique <- function() {
 #' df <- pl$DataFrame(a = c(1, 1, 2, 1))
 #' df$select((pl$col("a") == 1)$arg_true())
 expr__arg_true <- function() {
-  arg_where(self$`_rexpr`) |> 
+  arg_where(self$`_rexpr`) |>
```

Comment:
Please just ignore those fmt changes, they are tedious to fix and will be overwritten soon with Air.